src/Forms/Form.php

```diff
@@ -74,19 +74,33 @@ public function renderJavascript()
     }
     /**
      * (non-PHPdoc)
-     * @see \QuantumForms\FormInterface::validateInput()
+     * @see \QuantumForms\FormInterface::validate()
      */
-    public function validateInput(array $request)
```

Comment:
I thought the name is a bit long any cumbersome. I mean there's nothing else to validate other than input, right?